source/tutorials/fmt-cdiff-khanna.rst

```diff
@@ -400,6 +400,87 @@ For more information about animated ordinations, visit Emperor's `documentation`
 
 .. _`fmt cdiff taxonomy`:
 
+
+Longitudinal analysis
+---------------------
+
+For longitudinal studies, we can use the :doc:`q2-longitudinal plugin <longitudinal>` to evaluate the association between experimental observations, treatments, and time.
+
+We’ll start by using the ``first-distances`` method to examine how FMT reshapes an individual's gut microbiome to resemble that of the donor.
+
+.. command-block::
+
+  qiime longitudinal first-distances \
+    --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+    --m-metadata-file sample_metadata.tsv \
+    --p-state-column timepoint \
+    --p-individual-id-column donor_group \
+    --p-baseline -99 \
+    --p-replicate-handling random \
+    --o-first-distances core-metrics-results/unweighted_unifrac_first_distances.qza
+
+Next, we will use the ``volatility`` visualizer to interactively examine temporal changes in distance from donor, alpha diversity, and PCoA ordinations for each subject. We will drop two samples that were collected long before FMT treatment.
+
+.. command-block::
+
+  egrep -v '(10057.227.3|10057.227.2)' sample_metadata.tsv | grep 'atient' > sample_metadata_select.tsv
+
+  qiime longitudinal volatility \
+    --m-metadata-file sample_metadata_select.tsv \
+    --m-metadata-file core-metrics-results/unweighted_unifrac_first_distances.qza \
+    --m-metadata-file core-metrics-results/observed_otus_vector.qza \
+    --m-metadata-file core-metrics-results/unweighted_unifrac_pcoa_results.qza \
+    --p-state-column timepoint --p-default-group-column ibd_or_not \
+    --p-default-metric Distance \
+    --p-individual-id-column subject_code \
+    --o-visualization volatility.qzv
+
+
+In the interactive visualization, use the "Metric column" widget found in the toolbar on the right-hand side of the line plot to toggle between metrics displayed on the y-axis. Focus on "Distance" (UniFrac distance from donor), "observed_otus", and "Axis 1" (PCoA axis 1 ordinations). From these metrics, we see that 1) each subject's stool microbiota become more similar to the donor's after FMT; 2) FMT increases observed richness; and 3) FMT causes all samples to shift along PC axis 1, indicating a common shift in the microbial composition of those samples. We can test the significance of all of these effects using q2-longitudinal.
+
+The effects of FMT do not appear to be gradual, so we will bypass other approaches in q2-longitudinal for repeated measurements (see the :doc:`q2-longitudinal tutorial <longitudinal>` for more details on other methods in this plugin). Instead, we will focus just on changes in the microbiome pre- and post-FMT. We will use the ``pairwise-differences`` visualizer to perform Wilcoxon signed-rank tests of paired differences between each subject's samples pre- and post-FMT:

```

Comment:
Out of curiosity, is it possible to link to the specific section about repeated measures vs. the full tutorial?

Reply:
that's the remainder of the tutorial, so no.